src/main/java/com/rackspacecloud/metrics/ingestionservice/listeners/rawlisteners/RawListener.java

```diff
@@ -44,50 +56,48 @@ public RawListener(InfluxDBHelper influxDBHelper, MeterRegistry registry) {
             containerFactory = "batchFactory",
             errorHandler = "listenerErrorHandler"
     )
-    public boolean listenUnifiedMetricsTopic(
-            @Payload final List<Metric> records,
+    public void listenUnifiedMetricsTopic(
+            @Payload final List<ExternalMetric> records,
             @Header(KafkaHeaders.RECEIVED_PARTITION_ID) final int partitionId,
             @Header(KafkaHeaders.OFFSET) final long offset,
             final Acknowledgment ack) throws Exception {
 
+        registry.gauge("ingestion.records.count", Arrays.asList(rawListenerTag), gaugeRecordsCount);
```

Comment:
Could this be registered once in the constructor rather than doing a no-op register here?